lib/cert/runner.rb

```diff
@@ -44,6 +41,25 @@ def run
       end
     end
 
+    # Command method for the :revoke_expired sub-command
+    def revoke_expired_certs!
+      login
+
+      revoke_count = 0
+      expired_certs.each do |certificate|
+        begin
+          UI.message "#{certificate.id} #{certificate.name} has expired, revoking..."
+          certificate.revoke!
+          revoke_count += 1
+        rescue => e
+          UI.error "An error occurred while revoking #{certificate.id} #{certificate.name}"
+          UI.error "#{e.message}\n#{e.backtrace.join("\n")}" if $verbose
+        end
+      end
+
+      UI.success "#{revoke_count} expired certificate#{'s' if revoke_count != 1} #{revoke_count == 1 ? 'has' : 'have'} been revoked! 👍"
```

Comment:
We have that across the code base, maybe we should add a helper method to fastlane_core in the long run.